server/app.ts

```diff
@@ -0,0 +1,26 @@
+import { join } from 'path';
+import express, { Request, Response, Application } from 'express';
+import dotenv from 'dotenv';
+import cookieParser from 'cookie-parser';
+import compression from 'compression';
+
+const app:Application = express();
+dotenv.config();
+app.set('port', process.env.PORT || 5000);
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(cookieParser());
+app.use(compression());
+
+app.get('/', (req:Request, res:Response):void => {
+  res.json({ msg: 'Hello CleanStead' });
+});
```

Comment:
We should serve this route at the development only, and you should change the response body template from `msg` to `message`
```suggestion
if (NODE_ENV === 'development') {
  app.get('/', (req:Request, res:Response):void => {
    res.json({ message: 'Hello CleanStead' });
  });
}
```